syntax = "proto3";
package OnboardingLab3;

message SimpleMessage {
  optional string sender_name = 1;
  optional string sender_message = 2;
}

